common/annotations/bew.xml

```diff
@@ -15,47 +15,47 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="😀" type="tts" draft="unconfirmed">muka girang</annotation>
 		<annotation cp="😃" draft="unconfirmed">lèbar | muka | nyengir | seneng | sumringah</annotation>
 		<annotation cp="😃" type="tts" draft="unconfirmed">muka nyengir lèbar</annotation>
-		<annotation cp="😄" draft="unconfirmed">girang | ketawa | mata ketutup | muka | seneng</annotation>
+		<annotation cp="😄">girang | ketawa | mata ketutup | muka | muka girang mata ketutup | seneng</annotation>
```

Comment:
@DraganBesevic @pedberg-icu I think I see why this happens: `CLDRModify.CLDRFilter.replace` is supposed to be called with `fullpath` not just the path gotten by looping through the CLDRFile

Reply:
https://github.com/unicode-org/cldr/pull/3307